sylvia/tests/ui/attributes/features/invalid_params.stderr

```diff
@@ -0,0 +1,6 @@
+error: Invalid feature.
+       = note: Expected `#[sv::features(replies)];`.
```

Comment:
It's good to have a list of available features provided in the error message as it is implemented here. Nit: Instead of `Expected [...]` imo we could add `Available features: [feat1, feat2, ...]`. It's worth to note that features attribute probably could be used in few different contexts, so it's important to provide the list of features supported in a given place of use.

Reply:
Good point. Especially that this error is regarding the features not the attribute.